docker/run-tests-sp.sh

```diff
@@ -29,6 +29,13 @@ esac
 # shellcheck source=../scripts/fetch_envars.sh
 /test/scripts/fetch_envars.sh "${ENVIRONMENT}" | tee /test/.env
 
+if [ "${SAM_STACK_NAME:-}" == "authentication-api" ]; then
+  if grep ^API_RP_URL= /test/.env; then
+    API_RP_URL=$(grep ^API_RP_URL= /test/.env | cut -d= -f2)
```

Comment:
It's probably best to use `-f2-`, just in case the URL contains `=`

Reply:
Done